ui/src/main/java/org/pentaho/di/ui/job/entries/job/JobEntryJobDialog.java

```diff
@@ -218,15 +222,10 @@ public void widgetSelected( SelectionEvent arg0 ) {
       }
     } );
 
-    wbBrowse.addSelectionListener( new SelectionAdapter() {
-      public void widgetSelected( SelectionEvent e ) {
-        if ( rep != null ) {
-          selectJob();
-        } else {
-          pickFileVFS();
```

Comment:
Looks like pickFileVFS is now no longer called, which has additional logic . Also it appears to have an extra dialog which I'm not sure is required anymore. Will need to determine what is needed in that method